src/main/kotlin/org/rust/cargo/project/settings/ui/RustProjectSettingsPanel.kt

```diff
@@ -94,11 +94,11 @@ class RustProjectSettingsPanel(private val cargoProjectDir: String = ".") : Disp
 
     fun attachTo(layout: LayoutBuilder) = with(layout) {
         data = Data(
-            RustToolchain.suggest(),
-            true,
-            null,
-            false,
-            true
+            toolchain = RustToolchain.suggest(),
+            autoUpdateEnabled = true,
+            explicitPathToStdlib = null,
+            useCargoCheckForBuild = false,
+            useCargoCheckAnnotator = true
```

Comment:
By the way, it is absolutely horrendous that there is this super boring similar code in like six places! I definitely over engineered it. If you have any ideas on how to remove the duplication, please go ahead!